soc/espressif/esp32c2/memory.h

```diff
@@ -9,9 +9,9 @@
 #define SRAM0_SIZE         0x4000
```

Comment:
I think we should define SRAM0 in dts as IRAM only to have a single source of SRAM origins/sizes